src/VisualStudio/Core/Def/Implementation/ProjectTelemetry/IVisualStudioProjectTelemetryService.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectTelemetr
     /// <summary>
     /// In process service responsible for listening to OOP telemetry notifications.
     /// </summary>
-    internal interface IProjectTelemetryService : IWorkspaceService
+    internal interface IVisualStudioProjectTelemetryService : IWorkspaceService
```

Comment:
>: IWorkspaceService [](start = 60, length = 19)

I'd also remove `: IWorkspaceService`. Since these are all VS services they can MEF import `VisualStudioWorkspace` directly and can be retrieved directly from VS MEF composition.

Reply:
nice.

Reply:
done.